.github/workflows/test_conda-build.yml

```
@@ -18,19 +19,27 @@ jobs:
 
       - name: build conda env
         run: |
-            # The appended lines make sure that we actually build with Cylc
-            # too:
-            echo "  - pip" >> conda-evironment.yml
-            echo "  - pip": >> conda-environment.yml
-            echo "    - ." >> conda-environment.yml
-            cat conda-environment.yml
-            conda env create \
-              -f conda-environment.yml
-            . /usr/share/miniconda/etc/profile.d/conda.sh
+          # The appended lines make sure that we actually build with Cylc too:
+          echo "  - pip" >> conda-evironment.yml
+          echo "  - pip": >> conda-environment.yml
+          echo "    - ." >> conda-environment.yml
+          cat conda-environment.yml
```

There is a typo, but it looks like a duplicate anyway
```suggestion
          # The appended lines make sure that we actually build with Cylc too:
          echo "  - pip": >> conda-environment.yml
          echo "    - ." >> conda-environment.yml
          cat conda-environment.yml
```
Moreover, there's still this warning in the workflow run so is this even working?

>Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.

---

This should be correct as is.

https://stackoverflow.com/questions/67081266/conda-yml-file-prompts-please-add-an-explicit-pip-dependency

Could be a regression or an older version of Conda?

---

Ah ok, still a typo though:
```suggestion
          # The appended lines make sure that we actually build with Cylc too:
          # https://stackoverflow.com/a/67096191/3217306
          conda_env_file="conda-environment.yml"
          echo "  - pip" >> "$conda_env_file"
          echo "  - pip": >> "$conda_env_file"
          echo "    - ." >> "$conda_env_file"
          cat "$conda_env_file"
```